justfile

```diff
@@ -277,6 +277,7 @@ rctemplate := '''
 container_registry=""
 # Azure resource group/ resource name. Resource group will be created.
 azure_resource_group=""
+default_platform="AKS-CLH-SNP"
```

Comment:
This should go a view lines below, together with other things that don't need to be changed on initial setup.